Jenkinsfile

```diff
@@ -224,25 +224,87 @@ pipeline {
-                        runTests("test/unit/math/rev/fun/mdivide_left_tri_test.cpp")
-                        runTests("test/unit/math/rev/fun/multiply_test.cpp")
+                        script {
+                            if (isUnix()) {
+                                deleteDir()
+                                unstash 'MathSetup'
+                                sh "clinfo"
+                                sh "echo CXX=${env.CXX} -Werror > make/local"
+                                sh "echo STAN_OPENCL=true>> make/local"
+                                sh "echo OPENCL_PLATFORM_ID=0>> make/local"
+                                sh "echo OPENCL_DEVICE_ID=${OPENCL_DEVICE_ID}>> make/local"
+                                runTests("test/unit/math/opencl")
+                                runTests("test/unit/multiple_translation_units_test.cpp")
+                                runTests("test/unit/math/prim/fun/gp_exp_quad_cov_test.cpp")
+                                runTests("test/unit/math/prim/fun/mdivide_left_tri_test.cpp")
+                                runTests("test/unit/math/prim/fun/mdivide_right_tri_test.cpp")
+                                runTests("test/unit/math/prim/fun/multiply_test.cpp")
+                                runTests("test/unit/math/rev/fun/mdivide_left_tri_test.cpp")
+                                runTests("test/unit/math/rev/fun/multiply_test.cpp")
+                            } else {
+                                deleteDirWin()
+                                unstash 'MathSetup'
+                                bat "echo CXX=${env.CXX} -Werror > make/local"
+                                bat "echo STAN_OPENCL=true >> make/local"
+                                bat "echo OPENCL_PLATFORM_ID=1 >> make/local"
+                                bat "echo OPENCL_DEVICE_ID=0 >> make/local"
+                                bat 'echo LDFLAGS_OPENCL= -L"C:\\Program Files (x86)\\IntelSWTools\\system_studio_2020\\OpenCL\\sdk\\lib\\x64" -lOpenCL >> make/local'
+                                bat "mingw32-make.exe -f make/standalone math-libs"
+                                runTestsWin("test/unit/math/prim/fun/multiply_test.cpp", false, false)
+                                runTestsWin("test/unit/math/opencl", false, false)
+                                runTestsWin("test/unit/multiple_translation_units_test.cpp", false, false)
+                                runTestsWin("test/unit/math/prim/fun/gp_exp_quad_cov_test.cpp", false, false)
+                                runTestsWin("test/unit/math/prim/fun/mdivide_left_tri_test.cpp", false, false)
+                                runTestsWin("test/unit/math/prim/fun/mdivide_right_tri_test.cpp", false, false)                        
+                                runTestsWin("test/unit/math/rev/fun/mdivide_left_tri_test.cpp", false, false)
+                                runTestsWin("test/unit/math/rev/fun/multiply_test.cpp", false, false)
```

Comment:
Can you reorder these so that the same order is used for all combinations of windows/linux CPU/GPU?

Reply:
Done.